lighthouse-core/report/scripts/logger.js

```diff
@@ -67,4 +67,6 @@ class Logger {
   }
 }
 
-module.exports = new Logger('#log'); // singleton
```

Comment:
got rid of the nasty dom singleton. Viewer and the core report each defines their own logger now.